addons/web/static/src/views/basic_relational_model.js

```diff
@@ -263,6 +263,19 @@ export class Record extends DataPoint {
         return !this._invalidFields.size;
     }
 
+    openInvalidFieldsNotification() {
+        const invalidFields = [...this._invalidFields].map((fieldName) => {
```

Comment:
I would check here that there are invalid fields, to make this method a bit more robust (we never want a notification to be displayed if there's no invalid fields)